Gemfile

```diff
@@ -28,13 +28,14 @@ gem 'bootsnap', '>= 1.4.2', require: false
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
```

Comment:
Btw, do we need to specify the platforms here? I think we can remove that part from this gem.

Reply:
Thank you for this question! Seems like I misunderstood how to configure `.rubocop.yml`: that we should _disable_ rules we do not want and the rest is _enabled_ by default. Fixed it.

Reply:
About byebug and platforms:
> If a gem should only be used in a particular platform or set of platforms, you can specify them.

So seems like we can remove it. Will do it.